<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transf√©rer vers</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 10pt;
}
#liste_mat{
	
	background-color: #EEE;
	font: 10pt;
}

.QPushButton{
	border-radius: 2;
	background-color: qlineargradient(spread:pad, x1:0.548, y1:1, x2:0.469455, y2:0, stop:0.627119 rgba(242, 101, 101, 255), stop:1 rgba(255, 146, 117, 255));
	border-bottom: 2px solid;
	border-bottom-color:rgb(221, 83, 83);
	padding: 3px 16px;
	color:rgb(255, 221, 221);	
	font: 10pt;
}
.QPushButton:pressed{
	background-color: rgb(220, 91, 91);
	border-bottom-color: rgb(194, 73, 73);
}

.QLineEdit{
	border-radius: 2;
	border-bottom: 2px solid rgb(166, 166, 166);
	font: 10pt;
	padding: 3px 6px;
	
	background-color:rgb(222, 222, 222);
}

.QComboBox{
	border-radius: 2;
	background-color:rgb(222, 222, 222);
	border: none;
	padding: 3px 6px;
	border-bottom: 2px solid rgb(166, 166, 166);
}
.QComboBox::drop-down{
	 subcontrol-origin: padding;
    subcontrol-position: top right;
	
	border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
}
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}
QHeaderView::section {
	padding-left: 4px;
    border: 1px solid #6c6c6c;
}
QTableView {
	border-bottom: 1px solid #6c6c6c;
}
QTableView:!active {
    selection-background-color: palette(highlight);
    selection-color: palette(highlighted-text);
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="moyenDeTransportLabel">
       <property name="text">
        <string>Moyen de transport:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mdt">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nomDeChauffeurLabel">
       <property name="text">
        <string>Nom de chauffeur:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nomchauffeur">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choisir le projet destinataire:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="confirmer">
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
